src/StreetNameRegistry/Municipality/MunicipalityStreetName.cs

```diff
@@ -97,6 +98,47 @@ public void CorrectRetirement(Action<Names, HomonymAdditions, PersistentLocalId>
             Apply(new StreetNameWasCorrectedFromRetiredToCurrent(_municipalityId, PersistentLocalId));
         }
 
+        public void CorrectHomonymAdditions(
+            HomonymAdditions homonymAdditions,
+            Action<Names, HomonymAdditions, PersistentLocalId> guardUniqueActiveStreetNameNames)
```

Comment:
new lines

Reply:
It is one Action: Action<Names, HomonymAdditions, PersistentLocalId> guardUniqueActiveStreetNameNames)